File: src/ERP/acme.atena.domain/DTO/Account/PagamentoVendaFormaPagamento.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace acme.atena.domain.DTO.Account
{
    public class PagamentoVendaFormaPagamento : AbstractEntity
    {
        public Guid PagamentoVendaId { get; set; }
        public Guid FormaPagamentoId { get; set; }

        public virtual PagamentoVenda PagamentoVenda { get; set; }
        public virtual FormaPagamento FormaPagamento { get; set; }
    }
}

File: site/coreui-free-angular-admin-template-master/src/app/view-model/response/account/pagamento-venda-forma-pagamento.ts
import { AbstractEntity } from '../abstract-entity';
import { FormaPagamento } from './forma-pagamento';
import { PagamentoVenda } from './pagamento-venda';

export class PagamentoVendaFormaPagamento extends AbstractEntity{
  public pagamentoVendaId :string;
  public formaPagamentoId: string;

  public  pagamentoVenda: PagamentoVenda;
  public  formaPagamento: FormaPagamento;
}
